Add clarification about GitHub orgs to Sentry orgs (#1616)
getsentry/sentry-docs: src/collections/_documentation/workflow/integrations/global-integrations.md

    [{% asset github-global-add-repo.png %}]({% asset github-global-add-repo.png @path %})

> GitHub should now be enabled for all projects under your Sentry organization. Note that you can connect multiple GitHub organizations to one Sentry organization, but a single GitHub organization cannot be connected to multiple Sentry organizations.

Reviewer: ```suggestion
The GitHub integration is available for all projects under your Sentry organization. You can connect multiple GitHub organizations to one Sentry organization, but you **cannot** connect a single GitHub organization to multiple Sentry organizations.
```

> GitHub should now be enabled for all projects under your Sentry organization. Note that you can only have one GitHub organization integrated with one Sentry organization, but you can have multiple GitHub organizations connect with one Sentry organization.

Reviewer 2: This reads contradictory to me.

- First sentence says [GH org, Sentry org] _must_ have a  1:1 relationship;
- second sentence says [GH org, Sentry org] _may_ be N:1 (contradicting the first sentence).

Author: I see what you mean - updated to convey this better. Let me know if it's more clear.
